pytential/symbolic/primitives.py

```diff
@@ -1681,6 +1690,25 @@ def area(ambient_dim: int,
             cse_scope.DISCRETIZATION)
 
 
+@overload
+def mean(ambient_dim: int,
+         dim: int,
+         operand: ArithmeticExpression,
+         dofdesc: DOFDescriptorLike = None) -> ArithmeticExpression: ...
+
+@overload
+def mean(ambient_dim: int,
+         dim: int,
+         operand: ObjectArrayND[ArithmeticExpression],
+         dofdesc: DOFDescriptorLike = None) -> ObjectArrayND[ArithmeticExpression]: ...
+
+@overload
+def mean(ambient_dim: int,
+         dim: int,
+         operand: MultiVector[ArithmeticExpression],
+         dofdesc: DOFDescriptorLike = None) -> MultiVector[ArithmeticExpression]: ...
```

Comment:
Is there a way to do this without adding a bunch of overloads?

Reply:
Update: does exist.